htsohm/simulation/xenon_loading.py

```diff
@@ -0,0 +1,160 @@
+import sys
```

Comment:
Good extension of the pattern we established for creating new tests. Looks like the pattern worked!